CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # Changelog
 
-## 24.5.6 [#743](https://github.com/openfisca/openfisca-core/pull/743)
+## 24.6.0 [#744](https://github.com/openfisca/openfisca-core/pull/744)
```

Comment:
Ca ne devrait pas être une MAJOR plutôt ?

Reply:
What previous API call / usecase do you think would now break ? I didn't think of any, but maybe I missed something.

Reply:
So, I ssh'd into that build and looked for the `couple.json` file and it is indeed missing, which is the proximal cause of the tests failing.

Does this have to do with Pypi packaging and the JSON file maybe not being bundled up, does it have to be listed in a manifest file or something ?

Reply:
Exactly. And this is pretty hard to detect in the CI 🤕

Reply:
In the country-template, we avoid the issue by just packaging everything:
https://github.com/openfisca/country-template/blob/master/MANIFEST.in#L1

We may want to do the same in Core, but we should make sure it doesn't increase too much the size of the bundle.

Reply:
I tried to do the same I did in France with core but yaml test fail.

I think we have to refactor openfisca-run-test first, or rewrite the test_yaml.py.

Reply:
How do they fail? 
We don't really want to package YAML tests in the wheel anyway (users don't need the tests)